### ISSUE TITLE
Question regarding using ssl for docker container

### ISSUE THREAD

**Issue by harikt** [coturn/coturn#803]
Hi, 

I can run the docker coturn image via the command

```
docker run --name coturn -d --network=host \ 
    -v /etc/letsencrypt/live/coturn.site.com/privkey.pem:/letsencrypt/privkey.pem \
    -v /etc/letsencrypt/live/coturn.site.com/fullchain.pem:/letsencrypt/fullchain.pem \ 
    coturn/coturn -n --log-file=stdout \
    --min-port=49160 --max-port=49200 \
    --realm=coturn.site.com --cert=/letsencrypt/fullchain.pem --pkey=/letsencrypt/privkey.pem \
    --external-ip='$(detect-external-ip)' --relay-ip='$(detect-external-ip)' \ 
    --lt-cred-mech --fingerprint --listening-port=3478 --tls-listening-port=5349
```

Noticed the port 5349 was not listening. Checking the logs `docker logs coturn` i noticed the warning as below 

```
0: : WARNING: cannot find private key file: /letsencrypt/privkey.pem (1)
0: : WARNING: cannot find private key file: /letsencrypt/privkey.pem (1)
```

After login into the container I noticed there is a permission issue. The private key is not readable `/letsencrypt/privkey.pem`
 
![image](https://user-images.githubusercontent.com/120454/127375132-86ef6e98-94da-4777-874c-44df8bd7e218.png)

When passing `-v` I tried passing `:ro` like `-v /etc/letsencrypt/live/coturn.site.com/privkey.pem:/letsencrypt/privkey.pem:ro` still no luck.

Anyone has experienced similar issue, if so how can we fix this.

**Comment by tyranron** [comment]
@harikt the issue is not `coturn/coturn` Docker image specific and is well known. Similar questions may be easily googled:
- [Permission issue using volumes for SSL key/certificate with postgresql container](https://forums.docker.com/t/permission-issue-using-volumes-for-ssl-key-certificate-with-postgresql-container/70854/3)
- [Deploying postgresql docker with ssl certificate and key with volumes](https://stackoverflow.com/questions/55072221/deploying-postgresql-docker-with-ssl-certificate-and-key-with-volumes)

To sum up:
- either you change the permission for `privkey.pem` file to `604`;
- or you change the owner of the file to the uid of `nobody` user from the container;
- or you run your container as `--user=root`;
- or you create a custom Docker image on top of this one with certificates being embedded and permissions corrected.

Some container runtimes (as Kubernetes, for example) simplify this by providing addional options to configure mount permissions.

**Comment by harikt** [comment]
@tyranron thank you for the details.

I have been looking lots of places how to fix this.
